packages/tools/pie-icons-react/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "@justeattakeaway/pie-icons-react",
+  "version": "1.0.0-alpha.0",
+  "tag": "beta",
+  "main": "esm/index.js",
+  "files": [
+    "dist",
+    "esm",
+    "icons"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "sideEffects": false,
+  "homepage": "https://github.com/justeattakeaway/pie/tree/main/packages/tools/pie-icons-react",
+  "contributors": [
+    "Github contributors <https://github.com/justeattakeaway/pie/graphs/contributors>"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:justeattakeaway/pie.git"
+  },
+  "bugs": {
+    "url": "https://github.com/justeat/pie/issues"
+  },
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=14.0.0"
+  },
+  "keywords": [
+    "PIE"
+  ],
+  "scripts": {
+    "prepublishOnly": "yarn build",
+    "build:icons": "babel-node ./scripts/react-icons-gen.js",
+    "build:lib": "rollup -c",
+    "build": "yarn build:icons && yarn build:lib"
+  },
+  "devDependencies": {
+    "@babel/node": "7.19.1",
+    "@justeattakeaway/pie-icons": "2.0.0-beta.2",
+    "@svgr/core": "6.4.0",
+    "@types/react": "18.0.21",
+    "pascal-case": "3.1.2",
+    "react": "18.2.0",
+    "rollup": "2.56.2",
+    "rollup-plugin-delete": "2.0.0",
+    "rollup-plugin-typescript2": "0.30.0",
+    "typescript": "4.8.4"
```

Comment:
some of these dependencies might be able to be moved up to the project root `package.json` - I can imagine `typescript`, `rollup` and `@babel/node` being reused, for example :)

Reply:
Will look to do this as part of some tidy up PRs that will happen after we merge this if that's ok?